staging/src/k8s.io/apimachinery/pkg/api/meta/interfaces.go

```diff
@@ -72,6 +72,9 @@ type MetadataAccessor interface {
 	Continue(obj runtime.Object) (string, error)
 	SetContinue(obj runtime.Object, c string) error
 
+	ClusterName(obj runtime.Object) (string, error)
```

Comment:
This isn't necessary in this PR - I was going to use it for something, but didn't end up needing it. I can back it out, or we can keep it.

Reply:
We added this to meta.Accessor, so makes sense to have here as well